internal/webapi/server.go

```diff
@@ -62,13 +57,17 @@ func NewServer(db database.Database, auth authentication.Authenticator[APIAuthCr
 	femto.OnPost(mux, "/api/admin/add_workstation", api.addMachine)
 	femto.OnPost(mux, "/api/admin/stats/modify", api.modifyMachineInfo)
 	femto.OnPost(mux, "/api/admin/rm_workstation", api.removeMachine)
-	femto.OnGet(mux, "/api/admin/confirm", api.confirmAdmin)
+	femto.OnGet(mux, "/api/admin/confirm", func(r *http.Request, l *slog.Logger) (*femto.Response[UsernameReminder], error) {
+		return api.confirmAdmin(auth, r, l)
+	})
 
 	return &Server{mux, api}
 }
 
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Access-Control-Allow-Origin", "http://localhost:3000")
+	// TODO: Maybe unset in Caddyfile???
```

Comment:
It is **incredibly** important this is unset in the Caddyfile

Reply:
Unset in https://github.com/gpuctl/gpuctl/pull/140/commits/123d3bd7b78e32000b31a9032c69e9206809e4b0